### PR TITLE
feat: Edit code style

### PR DESCRIPTION
**Pull request by ianxinnew**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `tips` property in the `CodeConfigurator` component to provide contextual examples.
	- Enhanced `MetaCodeEditor` layout with a split-pane view for improved usability, making demo content always visible.
	- Expanded localization support with new entries for "example code" in English and Chinese.

- **Style**
	- Updated dialog box styling with rounded corners for a modern appearance.

### CODE REVIEW

File: packages/common/component/MetaCodeEditor.vue [hunk 1]
```diff
@@ -279,76 +283,86 @@ export default {
...
     cursor: pointer;
     color: var(--ti-lowcode-common-text-main-color);
   }
 }
+:deep(.tiny-dialog-box__header) {
+  padding-bottom: 15px;
+}
+
+:deep(.tiny-dialog-box__title) {
+  font-size: 14px;
```
Comment (wenmine) [inline packages/common/component/MetaCodeEditor.vue:310]: 这种 样式不是统一的吗，为啥要去调整

Comment (ianxinnew) [reply packages/common/component/MetaCodeEditor.vue:310]: 统一是12排序，这里12px标题字体太小，与设计沟通后改成14px

File: packages/common/component/MetaCodeEditor.vue [hunk 2]
```diff
@@ -20,28 +20,28 @@
...
-            <span>{{ editorState.showEditorDemo ? $t('common.collapseExample') : $t('common.expandExample') }}</span>
-            <icon-chevron-up v-if="editorState.showEditorDemo" class="collapse-icon"></icon-chevron-up>
-            <icon-chevron-down v-else class="collapse-icon"></icon-chevron-down>
-          </div>
         </div>
-        <div v-if="editorState.showEditorDemo" class="header-tips-demo">
-          <div class="header-tips-demo-content lowcode-scrollbar-thin">
-            <pre><code>{{ editorTipsDemo }}</code></pre>
-          </div>
+        <div class="demo-split-pane">
```
Comment (chilingling) [inline packages/common/component/MetaCodeEditor.vue]: 这个类名感觉不是很语义化？
- pane 应该改成 panel ?
- demo-split-pane 感觉不是很合适？叫 `code-editor-split-panel-wrapper` 或者是 `source-code-split-panel-wrapper` 会不会更语义化一点？

File: packages/common/component/MetaCodeEditor.vue [hunk 3]
```diff
@@ -20,28 +20,28 @@
...
+              <monaco-editor
+                ref="editor"
+                class="source-code-content"
+                :value="value"
+                :options="options"
+                @editorDidMount="editorDidMount"
+              ></monaco-editor>
+            </template>
+            <template #right>
+              <div class="source-code-split-panel-wrapper-right">
```
Comment (hexqi) [inline packages/common/component/MetaCodeEditor.vue]: image

没有示例代码时需要隐藏示例代码区域

Comment (ianxinnew) [reply packages/common/component/MetaCodeEditor.vue]: 重新提个pr优化

**Review by hexqi — DISMISSED** [review]

**Comment by hexqi** [comment]
怎么有这么多文件修改，合并主干有问题

**Review by hexqi — APPROVED** [review]

**Pull request merged**